python/src/Distribution.i

```diff
@@ -51,8 +60,9 @@ class SciPyDistribution(PythonDistribution):
         self._dist = dist
 
         # compute range
-        lb = dist.ppf(0.)
-        ub = dist.ppf(1.)
+        cdf_epsilon = 1e-12
```

Comment:
the computation should be done first using 0.0 and 1.0, and only the infinite bounds should be replaced by the value at 1.0e-12 or 1-1.0e-12. This way, you get the exact value for the finite bounds and a useful value for the infinite bounds